### PR TITLE
feat(cli): output query in YAML format

### PR DESCRIPTION
**Pull request by afiune**
## Summary

Since the Lacework CLI support YAML format, we need to have a way to present queries with that format,
this PR adds a new flag `--yaml` to output the query in YAML format.

## How did you test this change?

Added integration tests. 🚀

## Issue

Hackathon https://lacework.atlassian.net/browse/HACK-55

### CODE REVIEW

**Review by hazedav — CHANGES_REQUESTED** [review]
See comment pertaining to `YAMLOutput()` approach

File: cli/cmd/lql_show.go [hunk 1]
```diff
@@ -50,6 +58,18 @@ func showQuery(_ *cobra.Command, args []string) error {
 		return cli.OutputJSON(queryResponse.Data)
 	}
 
-	cli.OutputHuman(queryResponse.Data.QueryText)
+	if queryCmdState.YAML {
```
[the remark this answers is not in the document]
Comment (afiune) [reply cli/cmd/lql_show.go:61]: I don't think so, YAML is not a global flag, just like PDF is only available for compliance reports.

File: cli/cmd/lql_show.go [hunk 2]
```diff
@@ -46,10 +62,18 @@ func showQuery(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to show query")
 	}
+
 	if cli.JSONOutput() {
 		return cli.OutputJSON(queryResponse.Data)
 	}
 
+	if cli.YAMLOutput() {
+		return cli.OutputYAML(&api.NewQuery{
```
Comment (hazedav) [inline cli/cmd/lql_show.go:71]: nit. Were we not able to use `queryResponse.Data` here?  Was this simply an issue of `evaluatorID` being spit out?  If so would adding `omitempty` make sense?

**Review by hazedav — APPROVED** [review]

**Pull request merged**